src/render_helpers/shaders/color_interpol.frag

```diff
@@ -0,0 +1,42 @@
+
+vec4 color_linear(vec4 color) {
+  return vec4(
+    pow(color.r, 2.0),
+    pow(color.g, 2.0),
+    pow(color.b, 2.0),
+    color.a
+  );
+}
+
+vec4 color_root(vec4 color) {
+  return vec4(
+    sqrt(color.r),
+    sqrt(color.g),
+    sqrt(color.b),
+    color.a
+  );
+}
+
+vec4 color_mix(vec4 color1, vec4 color2, float color_ratio) {
+
```

Comment:
Could you format this file properly please? See how other shaders are formatted. Remove weird empty lines, add missing spaces, fix indentation, make the comments follow the same style.